source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -82,6 +82,26 @@ If your PSP is Stripe, you can sign in to the [GOV.UK Pay admin tool](https://se
 
 If your PSP is Government Banking's contracted PSP (currently Worldpay), SmartPay or ePDQ, contact your PSP to find out your minimum payout and payment times.
 
+#### Checking when your PSP sent a payment
+
+If your PSP is Stripe, you can check when Stripe sent a payment to your bank account by making an API call to [get information about a payment](/reporting/#get-information-about-a-single-payment).
+
+Example response:
+
+```
+"settlement_summary": {
+  ...
+  "settled_date": "2016-01-22",
```

Comment:
we should specify the timezone used for date cutoffs .. is it midnight UTC or midnight Local time?

Reply:
@sfount we should ideally have a unit test specifying this behaviour .. dates have timezones too!

Reply:
Great point @heathd, currently the dates are always provided by the PSP and should always be treated as UTC. This attempts to follow the pattern set by `captured_date` which uses the same format https://govukpay-api-browser.cloudapps.digital/#tocssettlementsummary

Reply:
Adding a comment to this effect.